orttraining/orttraining/core/session/training_session.h

```diff
@@ -245,6 +245,9 @@ class TrainingSession : public InferenceSession {
     // The pipeline configuration output.
     // This is only set if an pipeline is enabled.
     optional<PipelineConfigurationResult> pipeline_config_result;
+
+    // Mapped initilized names after weight partitioning for example MegatronTransformer
+    std::unordered_map<std::string, std::string> updated_weight_names{};
```

Comment:
this is not used, right?

Reply:
it is used to pass in to optimizer config generation